Assets/Creatures/api/control/InputManager.cs

```diff
@@ -40,7 +40,7 @@ private void Update()
 
         private void HandleMouseInput()
         {
-            Vector2 mousePosition = MousePosition;
+            Vector3 mousePosition = MousePosition;
```

Comment:
I had made it 2D because that made sense to me, but then when I went to bring in the existing shooting logic it expected a 3D mouse position ... so you tell me 🤣